packages/dds/tree/src/shared-tree/schematizeTree.ts

```diff
@@ -231,12 +211,14 @@ export function ensureSchema(
 /**
  * Content that can populate a `SharedTree`.
  */
-export interface TreeStoredContent {
+export interface TreeStoredContentStrict {
```

Comment:
Some tests still depend on the old options, so this was given a new name to avoid impacting them. The old format type has been moved into test utils.